src/caffe/layers/prelu_layer.cu

```diff
@@ -83,13 +88,13 @@ void PReLULayer<Dtype>::Backward_gpu(const vector<Blob<Dtype>*>& top,
     Dtype* slope_diff = this->blobs_[0]->mutable_gpu_diff();
     int cdim = channels * dim;
     Dtype dsum = 0.;
-    for (int n = 0; n < bottom[0]->num(); ++n) {
+
```

Comment:
I don't think you can simply remove the loop here. The `caffe_gpu_dot` and `caffe_gpu_gemv` needs to be in this loop.